codechecker_lib/analysis_manager.py

```diff
@@ -212,7 +212,7 @@ def signal_handler(*arg, **kwarg):
         pool.map_async(check,
                        analyzed_actions,
                        1,
-                       callback=worker_result_handler).get(float('inf'))
+                       callback=worker_result_handler).get()
```

Comment:
According to the docs, not passing an argument is the way to have no timeout. See https://docs.python.org/2/library/multiprocessing.html#multiprocessing.pool.AsyncResult.get